packages/manfred2html/src/engine/cv-monochrome-force/html-parts/studies-section/studies-section.ejs

```diff
@@ -0,0 +1,38 @@
+<%_ if (studiesCollection && studiesCollection.length !== 0) { -%>
+  <section class="section educations">
+    <h2 class="title"><%- labels.STUDIES_HEADING %></h2>
+    <%_ for (const study of studiesCollection) { -%>
+        <div class="education">
+          <div class="decoration">
+            <div class="decoration__circle"></div>
+            <hr class="decoration__line" />
+          </div>
+          <div class="education__group">
+            <p class="education__date"><%- new Date(study.startDate).toLocaleDateString(undefined, { year:"numeric", month:"short"})%> <%_ if(study.finishDate && study.finishDate.toLowerCase() !=='actualidad') { -%>
+              <%- "- "+new Date(study.finishDate).toLocaleDateString(undefined, { year:"numeric", month:"short"}) -%>
+            <%_ } else { -%> - Actualidad
```

Comment:
Create new label: CURRENT_DATE: 'string'

spanish -> CURRENT_DATE: 'Actualidad'
english -> CURRENT_DATE: 'now'

Reply:
It's done and it's also done in the experience section.